isa17hotsite/src/data/apoio.yml

```diff
@@ -23,4 +26,7 @@
     img: nextt.svg
   - name: Casa do Código
     url: https://www.casadocodigo.com.br/
-    img: cdc.svg
\ No newline at end of file
+    img: cdc.svg
+  - name: SoftExpert
+    url: https://www.softexpert.com.br/
+    img: softexpert.svg
```

Comment:
> No new line at the end of file

You should configure your code editor to save file with a blank line at the end of file ([Sublime](http://stackoverflow.com/questions/37794699/sublime-text-3-ensure-only-one-trailing-newline-at-end-of-file) example)